test/controllers/v1/bundles_controller_test.exs

```diff
@@ -136,6 +136,28 @@ defmodule Cog.V1.BundlesControllerTest do
             "version has already been taken"] = json_response(conn, 409)["errors"]
   end
 
+  test "Overwrites an installed version when bundle version enforcement is disabled", %{authed: requestor} do
+    Application.put_env(:cog, :enforce_bundle_version, false)
+
+    config = config(:map)
+    conn = api_request(requestor, :post, "/v1/bundles", body: %{"bundle" => %{"config" => config}})
+    assert response(conn, 201)
+
+    # Now try to do the same thing again
+    conn = api_request(requestor, :post, "/v1/bundles", body: %{"bundle" => %{"config" => config}})
+
+    bundle_version_id = Poison.decode!(conn.resp_body)
+                        |> get_in(["bundle_version", "id"])
+
+    bundle_version = Cog.Repository.Bundles.version(bundle_version_id)
+
+    assert conn.status == 201
+    assert bundle_version.config_file == config
+    assert bundle_version.bundle.name == config["name"]
+
+    Application.put_env(:cog, :enforce_bundle_version, true)
+  end
+
```

Comment:
Yeah, I agree. I just did a quick smoke test to validate it was working. If you think my general approach is reasonable, I'll add some more tests to make sure everything is working as expected.